src/auth/manager/service_test.go

```diff
@@ -32,19 +32,19 @@ var testManifest = []byte(`
   name: signer
   specs:
     permission:
-      - read:eth1accounts
+      - read:ethaccounts
```

Comment:
I guess this should be renamed to `read:ethereum`, right?

Reply:
lol why are these tests passing. Updated